src/style.css

```diff
@@ -0,0 +1,53 @@
+:root {
+  --height: 16px;
+  --width: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  height: var(--height);
+  width: var(--width);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+/* .stars__star:not(:last-child) {
+  margin: 0;
+} */
```

Comment:
If this causes your tests to fail, try to use only :last-child pseudo class

Reply:
tests end successfully